Source/Scene/ConditionsExpression.js

```diff
@@ -79,19 +87,27 @@ define([
         var runtimeConditions = [];
         var conditions = expression._conditions;
         if (defined(conditions)) {
-            var exp = expression._expression;
+            var expressions = expression._expressions;
             var length = conditions.length;
             for (var i = 0; i < length; ++i) {
                 var statement = conditions[i];
                 var cond = String(statement[0]);
                 var condExpression = String(statement[1]);
-                if (defined(exp)) {
-                    cond = cond.replace(expressionPlaceholder, exp);
-                    condExpression = condExpression.replace(expressionPlaceholder, exp);
-                } else {
-                    cond = cond.replace(expressionPlaceholder, 'undefined');
-                    condExpression = condExpression.replace(expressionPlaceholder, 'undefined');
+
+                //Loop over all expressions for replacement instead of only replacing one
```

Comment:
Shorten the description, just `Loop over all expressions for replacement` is fine.